templates/kubernetes/docs/explain-cloud.md

```diff
@@ -37,8 +37,9 @@ For more details, see the various cloud integration pages.
 <div class="p-notification--information">
   <div class="p-notification__content">
     <p class="p-notification__message">We appreciate your feedback on the documentation. You can
-    <a href="https://github.com/charmed-kubernetes/kubernetes-docs/edit/main/pages/k8s/explain-cloud.md" >edit this page</a>
+    <a href="https://github.com/charmed-kubernetes/kubernetes-docs/edit/main/pages/k8s/how-to-clouds.md" >edit this page</a>
```

Comment:
It looks like this was changed by mistake

Reply:
yup, well spotted :)

Reply:
fixed, I think. thanks for checking it over